frappe/core/page/permission_manager/permission_manager.js

```diff
@@ -286,7 +286,7 @@ frappe.PermissionEngine = Class.extend({
 					},
 					callback: function(r) {
 						r.message = $.map(r.message, function(p) {
-							return $.format('<a href="#Form/User/{0}">{1}</a>', [p, p]);
+							return $.format(p[0] + " - " + '<a href="#Form/User/{0}">{1}</a>', [p[1], p[1]]);
```

Comment:
Use string substitution instead

```suggestion
							return $.format(`${p[0]} - <a href="#Form/User/${p[1]}">${p[1]}</a>`);
```